### ISSUE TITLE
rrule(freq, count, byeaster) doesn't work with orthodox algorithm

### ISSUE THREAD

**Issue by torgash-ivanblch** [dateutil/dateutil#956]
1. `dateutils.easter.easter(2020)` returns Easter date calculated by EASTER_WESTERN algorithm. 
2. `dateutils.easter.easter(2020, 2)` returns one calculated by EASTER_ORTHODOX algorithm - and that satisfies me pretty well as it's what I actually need, but...
3. `dateutils.rrule.rrule(freq, count, byeaster)` could be extremely useful for generating multiple occurences of Easter and Easter-related holidays if I could somehow change its default behaviour to orthodox algorithm, as it uses western by default. 
Is it possible?

**Comment by pganssle** [comment]
I always hate to add things that our outside the spec, but given that we have `BYEASTER` already it seems fair to allow the use of other types of easter.

However, the original `rrule` parser was designed before [RFC 5545](https://tools.ietf.org/html/rfc5545) and its various extensions. Specifically I'm wondering if [RFC 7529](https://tools.ietf.org/html/rfc7529) might be relevant. I haven't looked at this in detail, but two open questions that I think would be good for this:

1. Is "orthodox easter" equivalent to "easter calculated on the Julian calendar"? If so, would support for Julian rrules get us this "for free"?
2. Is there any way to re-formulate `byeaster` in terms of standard recurrences available in any of the extensions to the original RFC? If so, we can maybe make it an alias for one of those.

If (and I suspect this is likely) we don't get an "easy out" from one of the above lines of inquiry, the main question is the UI for this. Should we have a wrapper object similar to the weekday objects like `rrule.EASTER(offset, calendar=EASTER.WESTERN)`? Should we just accept a tuple like `byeaster=(3, easter.EASTER_ORTHODOX)`? And finally, how do we serialize this with `str()` and deserialize it with `rrulestr`?

**Comment by torgash-ivanblch** [comment]
>     1. Is "orthodox easter" equivalent to "easter calculated on the Julian calendar"? If so, would support for Julian rrules get us this "for free"?
No. You can check that by calculating easter for different years with methods 1 and 2, they are not even simply some days offset from each other, it's complicated.
